deploy/kubernetes/tracee.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: tracee
+  name: tracee
+spec:
+  selector:
+    matchLabels:
+      app: tracee
+  template:
+    metadata:
+      labels:
+        app: tracee
+      name: tracee
+    spec:
+      containers:
+      - image: aquasec/tracee:latest
+        imagePullPolicy: Always
+        env:
+        - name: WEBHOOK_URL
+          value: http://webhook:2801
+        args:
+          - ./tracee-ebpf --output=format:gob | ./tracee-rules --input-tracee=file:stdin
+            --input-tracee=format:gob --webhook $WEBHOOK_URL --webhook-template /templates/falcosidekick.tmpl --webhook-content-type application/json
+        command:
+          - /bin/sh
+          - -c
+        ports:
+        - containerPort: 8096
+          protocol: TCP
+        name: tracee
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - name: tmp-tracee
+          mountPath: /tmp/tracee
+        - name: usr-src
+          mountPath: /usr/src
+          readOnly: true
+        - name: lib-modules
+          mountPath: /lib/modules/
+          readOnly: true
+        - name: tracee-templates
+          mountPath: /templates
```

Comment:
we will start adding the template to the container since #683 so will not be necessary